src/Tribe/Views/V2/Template_Bootstrap.php

```diff
@@ -423,7 +424,7 @@ public function filter_add_body_classes( $classes ) {
 			return $classes;
 		}
 
-		$classes[] = 'page-template-' . sanitize_title( $active_theme );
+		$classes[] = 'page-template-' . sanitize_title( (string) $active_theme );
```

Comment:
I think we should be more explicit here and "show our work" but I can be overruled.
```suggestion
		$classes[] = 'page-template-' . sanitize_title( (string) $active_theme->display( 'Name' );
```

Reply:
[wp_get_theme](https://github.com/WordPress/wordpress-develop/blob/6.6.2/src/wp-includes/theme.php#L117-L134) is _always_ going to return an object

Reply:
When a class has a `__toString()` method, it handles being cast to a string. WP_Theme has that method [here](https://developer.wordpress.org/reference/classes/wp_theme/__tostring/).